trpcultivate_phenotypes/src/TripalCultivateValidator/TripalCultivatePhenotypesValidatorBase.php

```diff
@@ -201,4 +220,41 @@ public function getConfigAllowNew() {
 
     return $allownew;
   }
+
+  /**
+   * Split or explode a data file line/row values into an array using a delimiter.
+   * 
+   * @param string $row
+   *   A line in the data file.
+   * 
+   * @return array
+   *   An array containing the values extracted from the line after splitting it based
+   *   on a delimiter value.
+   */
+  public static function splitRowIntoColumns(string $row) {
+    // Delimiter:
+
+    // @TODO: use the delimiter getter.
+    $delimiter = "\t";
+    if (empty($delimiter)) {
+      throw new \Exception('No delimiter provided.');
+    }
+    
+    // Split the values.
+    $values = explode($delimiter, $row);
```

Comment:
I think we're actually better to use str_getcsv() here. The delimiter can be passed into str_getcsv() as the separator. str_getcsv() differs from explode() in that it supports files with quotes around columns and escape charaters to allow quotes/delimiter in the columns.